src/collections/aggregate/index.ts

```diff
@@ -422,7 +451,7 @@ class AggregateManager<T> implements Aggregate<T> {
   base(
     metrics?: PropertiesMetrics<T>,
     filters?: FilterValue,
-    groupBy?: (keyof T & string) | GroupByAggregate<T>
+    groupBy?: (keyof T & string) | GroupByAggregate<T> | string
```

Comment:
Is this `groupBy` meant to be of a different type from the ones above?

```ts
groupBy: T extends undefined ? string : (keyof T & string) | GroupByAggregate<T> // aka PropertyOf
```

If my understanding is correct, both should have the same effect.

Reply:
> both should have the same effect.

If the above's true and we agree on adding `PropertyOf`, could we use it here as well?

Reply:
sure, will do!